tasks/main.yml

```diff
@@ -12,10 +12,32 @@
         "
 
 - name: Create a temp directory to store files needed by the run
+  notify: delete temp items
   tempfile:
     state: directory
     prefix: "pantheon-deploy-"
   register: _run_temp_dir
+  when:
+    - pantheon_deploy.target.build_dir | default('') == ''
+
+- name: Check for defined build directory
+  stat:
+    path: "{{ pantheon_deploy.target.build_dir }}"
+  register: _run_build_dir_stat
+  when:
+    - pantheon_deploy.target.build_dir | default('') != ''
+
+- name: Create the defined directory to store files needed by the run
+  file:
+    state: directory
+    path: "{{ pantheon_deploy.source.build_dir }}"
```

Comment:
shouldn't this be `.target.` and not `.source.`?

Reply:
Now it shows up, that took a while on Github's end, fixed!